### PR TITLE
Era modifier updated from Run3 to Run3_2024 in dqm client configs

### PR DESCRIPTION
**Pull request by jaisatul**
#### PR description:

This PR modifies the era in the DQM client config files to fix the data emulator mismatches seen in BMTF Muon quality plots.

#### PR validation:

Code quality check performed. Changes under test with DQM Online streamers

### CODE REVIEW

**Comment by missirol** [comment]
@cms-sw/dqm-l2 

What's the plan for this PR ? Sounds like it has some relevance for online operations (fix for spurious warnings in L1T online-DQM).

File: DQM/Integration/python/clients/l1tstage2_dqm_sourceclient-live_cfg.py [hunk 1]
```diff
@@ -1,8 +1,8 @@
 import FWCore.ParameterSet.Config as cms
 
 import sys
-from Configuration.Eras.Era_Run3_cff import Run3
-process = cms.Process("L1TStage2DQM", Run3)
+from Configuration.Eras.Era_Run3_2024_cff import Run3_2024
+process = cms.Process("L1TStage2DQM", Run3_2024)
```
Comment (missirol) [inline DQM/Integration/python/clients/l1tstage2_dqm_sourceclient-live_cfg.py:5]: Just out of curiosity, why is this change needed in this client ? (as opposed to changing only `l1tstage2emulator_dqm`)

**Comment by tjavaid** [comment]
> @cms-sw/dqm-l2
> 
> What's the plan for this PR ? Sounds like it has some relevance for online operations (fix for spurious warnings in L1T online-DQM).

I see the PR in draft mode. @jaisatul , do we expect another commit or is it ready for a review ?

File: DQM/Integration/python/clients/l1tstage2_dqm_sourceclient-live_cfg.py (hunk 1 above)
Comment (eyigitba) [reply DQM/Integration/python/clients/l1tstage2_dqm_sourceclient-live_cfg.py:5]: I think this is not strictly necessary, but if it doesn't change the functionality I would apply the change. It'll be required if we implement something like this to L1T DQM clients

**Comment by tjavaid** [comment]
please test

**Comment by mmusich** [comment]
@cmsbuild ignore tests-rejected with ib-failure

**Comment by mmusich** [comment]
@cms-sw/dqm-l2 can this be signed, given tests at P5 (see https://github.com/cms-sw/cmssw/pull/45242#issuecomment-2304369160) are OK?

**Comment by tjavaid** [comment]
+1

**Comment by mandrenguyen** [comment]
+1
